pkg/segment/segment.go

```diff
@@ -300,3 +303,11 @@ func sanitizeExec(errString string) string {
 	errString = pattern.ReplaceAllString(errString, fmt.Sprintf("exec command %s", Sanitizer))
 	return errString
 }
+
+func GetDebugTelemetry() string {
```

Comment:
Could you rename it to `GetDebugTelemetryFile` or similar, this name is misleading, I thought it was about telemetry enabled/disabled

Reply:
updating